### ISSUE TITLE
statistics/members shows charts twice when Infrastructure:All is selected

### ISSUE THREAD

**Issue by schadom** [inex/IXP-Manager#448]
##### ISSUE TYPE

Bug Report

##### OS

```
root@ixpmanager:~# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.5 LTS
Release:	16.04
Codename:	xenial
```

##### VERSION

```
root@ixpmanager:~# cat /srv/ixpmanager/library/IXP/Version.php | grep APPLICATION
define( 'APPLICATION_VERSION', '4.8.0' );
define( 'APPLICATION_VERDATE', '201807030000' );
```

##### ENVIRONMENT 

```
root@ixpmanager:~# php -v
PHP 7.0.31-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: Jul 25 2018 10:00:37) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.0.31-1+ubuntu16.04.1+deb.sury.org+1, Copyright (c) 1999-2017, by Zend Technologies
```

```
root@ixpmanager:~# dpkg -l | grep php7.0 | awk '{print $2}'
libapache2-mod-php7.0
php7.0
php7.0-bcmath
php7.0-cgi
php7.0-cli
php7.0-common
php7.0-curl
php7.0-gd
php7.0-intl
php7.0-json
php7.0-mbstring
php7.0-mcrypt
php7.0-mysql
php7.0-opcache
php7.0-readline
php7.0-snmp
php7.0-xml
php7.0-zip
```

##### CONFIGURATION

```
root@ixpmanager:/srv/ixpmanager# cat .env | egrep -v '(^#|^\s*$|^DB_|^APP_KEY|^HELPDESK|^IDENTITY|^MAIL_|^IXP_API_RIR_PASSWORD|^IXP_API_PEERING_DB_)'
APP_DEBUG=false
APP_URL="https://ixp.myixp.net"
APP_TIMEZONE="Europe/Vienna"
APP_LOG="single"
IXP_API_JSONEXPORTSCHEMA_PUBLIC=true
IXP_MULTIIXP_ENABLED=false
IXP_RESELLER_ENABLED=false
IXP_AS112_UI_ACTIVE=true
GRAPHER_BACKENDS="mrtg"
GRAPHER_BACKEND_MRTG_WORKDIR="/srv/mrtg"
GRAPHER_BACKEND_MRTG_LOGDIR="/srv/mrtg"
CACHE_DRIVER=memcached
SESSION_DRIVER=file
DOCTRINE_PROXY_AUTOGENERATE=true
DOCTRINE_CACHE=memcached
DOCTRINE_CACHE_NAMESPACE=IXPMANAGERNAMESPACE
IXP_IRRDB_BGPQ3_PATH="/usr/bin/bgpq3"
```

##### SUMMARY

IXP-Manager shows a single members mrtg chart twice under /statistics/members when `Infrastructure: All` is chosen. This does not occur if the dropdown value is changed to `Infrastructure: OurIXP` and the "Show graphs" button is triggered. We only have one single infrastructure defined in `infrastructure/list`.

##### STEPS TO REPRODUCE

See above.

##### EXPECTED RESULTS

IXP-Manager should show the same mrtg chart outputs for both `Infrastructure: All` and `Infrastructure: OurIXP`, because we only have one infrastructure configured in `infrastructure/list`.

##### ACTUAL RESULTS

See above.

##### IMPORTANCE

Low

**Comment by schadom** [comment]
Might be related to #376 . We do have some customers with member status "Not connected" and port status "Awaiting XConnect" as well.

**Comment by digitalandrei** [comment]
We also have this issue. We are running the latest snapshot.

**Comment by barryo** [comment]
Fixed in #480
